### PR TITLE
Exclude tests

### PR DESCRIPTION
**Pull request by tatraian**
- Windows platforms doesn't supports C++11 (using typedef is only in VS 14 - 2015)
- Unix errors come from c++0x missing header and OpenSSL link error

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -51,6 +51,24 @@ env:
 
 matrix:
   exclude:
+    # Lack of required C++11 level. Default uses only c++0x
+    - os: linux
+      env: PROJECT_DIR=examples/msgpack TOOLCHAIN=default
+
+    # I'dont know what is the problem, build fails with error messages:
+    # clang: error: argument unused during compilation: '-pie'
```
Comment (ruslo) [inline .travis.yml:61]: Looks like a warning. @tatraian please add link to logs of broken builds.

Comment (tatraian) [inline .travis.yml:61]: Yes. This is default:
https://travis-ci.org/ingenue/hunter/jobs/127172996
and this is the clang "error":
https://travis-ci.org/ingenue/hunter/jobs/127173004

**Comment by tatraian** [comment]
I think I sent the pr to wrong branch... Example is missing from hunter. (Or I just forget commit the example). I try to fix it tomorrow. I think I have to send PR to pkg.msgpack-406 branch.

**Comment by ruslo** [comment]
> I have to send PR to pkg.msgpack-406 branch

No, you need to push new changes to the branch `tatraian:pkg.msgpack`. They will be visible here automatically. Links to the build logs may came from `*.pr-406` temporary branch, it's okay.

File: .travis.yml (hunk 1 above)
Comment (ruslo) [inline .travis.yml:61]: > Yes. This is default

Okay, please add them to the `.travis.yml` file comments.

**Comment by tatraian** [comment]
I merged the examples and cmake files to pkg.msgpack branch. And I added the base tests to the corresponding files.

**Comment by ruslo** [comment]
> And I added the base tests to the corresponding files

No need, change need to contain only `.travis.yml` and `appveyor.yml` patches.

**Comment by tatraian** [comment]
That test ran last time. Now I get the following error:

```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

The build has been terminated
```

Is this not the glitch of Travis as well?

**Comment by ruslo** [comment]
> Is this not the glitch of Travis as well?

It depends, in this case yes. I've kicked build again: https://travis-ci.org/ingenue/hunter/jobs/127801479

**Comment by tatraian** [comment]
As I see it works now!

**Comment by ruslo** [comment]
> As I see it works now!

Okay, cool. But anyway only `.travis.yml`/`appveyor.yml` patches expected in `ingenue/hunter`. Flow is next: `develop` -> `pkg.template` -> `pkg.msgpack` so if you merge new stuff to `pkg.msgpack` directly it may lead to tons of conflicts. It's okay if this patch will not result to successful build.

**Comment by tatraian** [comment]
Thats ok, but the whole msgpack-c is not part of hunter now (in contrast of websocketpp). So I cannot send you test branch bases of hunter's msgpack code until you don't merge msgpack pr into hunter.

**Comment by ruslo** [comment]
>  So I cannot send you test branch

Why not? You can create branch based on `ingenue/pkg.msgpack` then apply `.travis.yml`/`appveyor.yml` patches and open pull request.

**Comment by tatraian** [comment]
I did this. But that branch doesn't contain msgpack codes... You created an other branch depending on msgpack hunter pr (pkg.msgpack-pr-406) , it contains that changes. But the original doesn't. 
I think this is since msgpack is not merged "officially" into hunter.

**Comment by ruslo** [comment]
> I think this is since msgpack is not merged "officially" into hunter

It doesn't matter, here is what I mean: https://github.com/ingenue/hunter/pull/15

**Comment by tatraian** [comment]
Sorry. I really don't understand what do you like. What I do? You have an `pkg.msgpack` branch that doesn't contain examples and hunter files for msgpack. How would you like how I put it in it?

**Comment by ruslo** [comment]
> How would you like how I put it in it?

Later

As I said:
- flow is `develop` -> `pkg.template` -> `pkg.msgpack`
- `ingenue/hunter` is for contribuing `.travis.yml`/`appveyor.yml`: https://github.com/ingenue/hunter/blob/pkg.template/README.rst
- it's okay if build **here** failed because there is no examples for now

Will happens like this:
- you send pull request to `ingenue/hunter@pkg.msgpack` with disabling toolchains patch -> I accept it (build may fail)
- you send pull request to `ruslo/hunter@develop` with new `msgpack` package -> I'm creating special branch for testing in `ingenue/hunter@pkg.msgpack.pr-N` -> all tests should pass at this point
- after patch merged to `ruslo/hunter@develop` branch then it goes to `pkg.template` -> `pkg.msgpack`, then I'm creating branch `upload.msgpack` for uploading binaries

**Comment by ruslo** [comment]
Integrated as https://github.com/ingenue/hunter/pull/15

**Comment by tatraian** [comment]
Thanks. Sorry I went home at Friday...

**Pull request closed without merging**